app/models/vso_appointment.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class VsoAppointment
```

Comment:
minor but you can use ActiveSupport::Inflector to allow acronyms like 'VSO' so you can have `VSOAppointment`
https://github.com/department-of-veterans-affairs/vets-api/blob/e7172a1cae43a81da310940435cb6bac86c746fe/config/initializers/inflections.rb#L8